Resources/public/js/extensions/ez-draftconflictbutton.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-draftconflictbutton', function (Y) {
+    "use strict";
+    /**
+     * The draft conflict button extension
+     *
+     * @module ez-draftconflictbutton
+     */
+    Y.namespace('eZ');
+
+
+    /**
+     * Views extension providing the draft conflict feature to an edit button
+     *
+     * When a view is extended with this extension it needs to add an event on the button that
+     * calls `_fireEditContentRequest` when fired.
+     * Also, the `_getContentEditRequestParams` method need to be implemented in the view.
+     *
+     * container scales to match the viewport's height new dimension.
+     *
+     * @namespace eZ
+     * @class HeightFit
+     * @extensionfor Y.View
+     */
+    Y.eZ.DraftConflictButton = Y.Base.create('draftConflictButtonExtension', Y.View, [], {
+
+        /**
+         * Provides the parameters needed to the content edit request
+         *
+         * @method _getContentEditRequestParams
+         * @param {Y.Node} target the target that has been clicked
```

Comment:
it's not necessarily a click so I would document that as: *target element of the event triggering the edit request*